pkg/apis/extensions/v1beta1/types.go

```diff
@@ -175,7 +175,7 @@ type ThirdPartyResourceData struct {
 	Data []byte `json:"data,omitempty"`
 }
 
-// Deployment enables declarative updates for Pods and ReplicationControllers.
+// Deployment enables declarative updates for Pods and ReplicationSets.
```

Comment:
Lets not touch deployment comments, till we have updated the code.
Please revert changes to deployment comments